client/src/components/MUI_NAVBAR.js

```diff
@@ -145,13 +145,13 @@ const MUI_NAVBAR = () => {
               id="menu-appbar"
               anchorEl={anchorElNav}
               anchorOrigin={{
-                vertical: 'bottom',
-                horizontal: 'left',
+                vertical: "bottom",
+                horizontal: "left",
               }}
               keepMounted
               transformOrigin={{
-                vertical: 'top',
-                horizontal: 'left',
+                vertical: "top",
+                horizontal: "left",
```

Comment:
@vmaineng @JosephThawn just curious, what changed when using double quotes?

Reply:
The reason why this changed is when I had to resolve merging conflicts, I accepted the incoming changes. which included the double quotes instead of single.

Reply:
@vmaineng ahhh ok ok, sounds good, thanks for addressing that :)